src/libs/Navigation/types.ts

```diff
@@ -200,6 +200,10 @@ type SettingsNavigatorParamList = {
         categoryName: string;
         backTo?: Routes;
     };
+    [SCREENS.WORKSPACE.CATEGORY_GLCODE]: {
+        policyID: string;
+        categoryName: string;
```

Comment:
I'm noticing the other workspace category pages have `backTo` so we might want to allow that here?

Reply:
Or maybe that's cuz we're linking to category pages from other places in the app, where we probably won't link to this GL Code edit page from other pages in the app? maybe?

Reply:
Yes exactly! I don't see us linking the `gl code` page from somewhere else. In any case, I think we should add this feature only when it's actually being used somewhere.

Reply:
agreed, ya good plan 👍